internal/driver/webhtml.go

```diff
@@ -491,6 +584,17 @@ function dotviewer(nodes) {
   }
 
   function setBackground(elem, set) {
+    // Handle table row highlighting.
+    if (elem.nodeName == "TR") {
+      if (set) {
```

Comment:
depending if not supporting IE10 is not an issue (Edge works fine), this can be simplified to
elem.classList.toggle("hilite", set)